src/Parser.php

```diff
@@ -43,8 +44,6 @@ class Parser
     private $tags = [];
     private $languageSpecifierLine;
 
-    private $passedNodesStack = [];
```

Comment:
This was only used to track recursion into nodes for the purpose of making assumptions about whether we could take tags (and looks like it wasn't always unwound). This is now handled differently.